front/src/App.vue

```diff
@@ -1,31 +1,30 @@
 <template>
-  <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link>
-    </div>
-    <router-view/>
-  </div>
+  <v-app id="app" light>
+    <component :is="layout">
+      <router-view/>
+    </component>
+  </v-app>
 </template>
 
-<style>
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-}
+<script>
+// @ is an alias to /src
+import { VApp } from 'vuetify/lib';
 
-#nav {
-  padding: 30px;
-}
+const defaultLayout = 'default';
 
-#nav a {
-  font-weight: bold;
-  color: #2c3e50;
-}
+export default {
+  name: 'App',
+  components: {
+    VApp
+  },
+  computed: {
+    layout () {
+      return (this.$route.meta.layout || defaultLayout) + '-layout';
```

Comment:
i am not sure that it would be good decision, but we can try and see

Reply:
I left this decision for now